### DISCUSSION TITLE
Used an internal xpub in BTC Pay Server, unable to access funds

### DISCUSSION THREAD

**Discussion by jimmydjabali** [lightningnetwork/lnd#7033] [Q&A] [answered]
Hello,

Recently I have set up a BTC Pay server and wanted to use my LND's xpub.
When doing `lncli wallet accounts list`, there is tons of accounts, and in my BTCPay Server I used the xpub with the `m/1017'/0'/0'` **derivation_path**.

I now learn that it's an "internal" LND account, but I already received few payments on this xpub, and unfortunately I cannot access them in any way.

Do you have an idea of how I can recover funds received on this special, LND's internal xpub ? (Native Segwit's format)

Thanks a lot for your time :pray:

**Comment by guggero** [comment, marked as answer]
You can use [`chantools genimportscript --derivationpath "m/1017'/0'/0'"`](https://github.com/guggero/chantools/blob/master/doc/chantools_genimportscript.md) to generate the private keys from the seed and then import it either into Bitcoin Core or Electrum (see `--format`).

**Reply by jimmydjabali** [reply]
Thanks a lot! It works.
Just sent you a few sats for the help.
